Base version:
* https://github.com/szatanjl/dwm/commit/1529909466206016f2101457bbf37c67195714c8

Notes:
* Fix transparant borders when picom is active.

--- a/drw.c
+++ b/drw.c
@@ -203,6 +203,8 @@ drw_clr_create(Drw *drw, Clr *dest, const char *clrname)
 	                       DefaultColormap(drw->dpy, drw->screen),
 	                       clrname, dest))
 		die("error, cannot allocate color '%s'", clrname);
+
+	dest->pixel |= 0xff << 24;
 }

 /* Wrapper to create color schemes. The caller has to call free(3) on the
